### PR TITLE
Set sidebar link hrefs

### PR DESCRIPTION
**Pull request by jwalgran**
Links provided by Parks.

### CODE REVIEW

**Comment by jwalgran** [comment]
@steventlamb rebased with a late-breaking change from Parks

File: src/nyc_trees/apps/core/templates/core/partials/nav.html [hunk 1]
```diff
@@ -36,8 +36,10 @@
     <div class="nav-side-sub">
         <h3>NYC</h3>
         <h6>Copyright 2015, The City of New York</h6>
-        <a href="#">Contact Us</a>
-        <a href="#">Privacy Policy</a>
+        <a title="treescount.help@parks.nyc.gov" href="mailto:treescount.help@parks.nyc.gov">Contact Us</a>
```
Comment (hectcastro) [inline src/nyc_trees/apps/core/templates/core/partials/nav.html]: We have this e-mail address as `DEFAULT_HELP_EMAIL` in the settings file if we want to reuse it.

**Comment by steventlamb** [comment]
+1 links work

File: src/nyc_trees/apps/core/templates/core/partials/nav.html (hunk 1 above)
Comment (jwalgran) [inline src/nyc_trees/apps/core/templates/core/partials/nav.html]: It's embedded in emails too. Building a context processor and DRYing out the templates is a good idea but a little more than I want to tackle right now.

**Pull request merged**